### ISSUE TITLE
Selected value on select

### ISSUE THREAD

**Issue by MasterPuffin** [dy/sprae#50]
When I have a select like this
```
 <select class="form-control" :name="field.name" value="object[field.name]">
    <option :each="option in field.options" value="option.value" text="option.label" ></option>
</select>
```
then there is no option wich has selected applied to it and the first option is selected by the browser. When using the following code, the current value from the object is properly preselected:
```
 <select class="form-control" :name="field.name" value="object[field.name]">
    <option :each="option in field.options" value="option.value" text="option.label" :selected="object[field.name] === option.value"></option>
</select>
```
Is this workaround intended? I would expect to have the correct value displayed by default.

**Comment by dy** [comment]
@MasterPuffin thanks for the report, fixed in v10.12.0. Now select and select multiple display selected options properly. Please lmk if it works.

**Comment by MasterPuffin** [comment]
Hi, 
I tested it and it now works as intended.
Thank you :smile:
